cylc/flow/scripts/show.py

```diff
@@ -307,12 +307,12 @@ async def prereqs_and_outputs_query(
                     prereqs.append([
                         True,
                         '',
-                        item['expression'].replace('c', ''),
+                        item['expression'],
                         item['satisfied']
                     ])
                 for cond in item['conditions']:
                     if multi_cond and not options.list_prereqs:
-                        prefix = f'\t{cond["exprAlias"].strip("c")} = '
```

Comment:
Gotta leave this in for now I'm afraid.

Reply:
Ah right, forgot that `cylc show` needs the workflow in question to be running (which might be on an older version)